src/JBPermissions.sol

```diff
@@ -224,16 +224,14 @@ contract JBPermissions is IJBPermissions {
     /// @notice Converts an array of permission IDs to a packed `uint256`.
     /// @param permissionIds The IDs of the permissions to pack.
     /// @return packed The packed value.
-    function _packedPermissions(uint256[] calldata permissionIds) internal pure returns (uint256 packed) {
+    function _packedPermissions(uint8[] calldata permissionIds) internal pure returns (uint256 packed) {
         // Keep a reference to the permission being iterated on.
         uint256 permissionId;
```

Comment:
could reduce the size in memory here?

Reply:
not sure here. i reduced it to make it easier to pass data around from the struct